### PR TITLE
Increase test coverage for `/modules/ui`

### LINKED ISSUE #3009: Increase test coverage for `/modules/ui`
## Scope & Context

This is a child issue of #2992 which addresses increasing test coverage for the front-end. This issue is for the increasing coverage for `/modules/ui`

### PR DESCRIPTION
**Pull request by gitstart-twenty**
### COVERAGE
Screenshot 2024-01-08 at 20 29 30

Fixes #3009

### CODE REVIEW

**Review by charlesBochet — APPROVED** [review]

**Review by charlesBochet — APPROVED** [review]
Great work, thank you @gitstart-twenty

File: packages/twenty-front/jest.config.js [hunk 1]
```diff
@@ -33,7 +35,12 @@ export default {
     'generated-metadata/*',
     'generated/*',
     '__stories__/*',
-
+    'utilities/recoil-scope/utils/*',
+    'modules/ui/theme/utils/themeColorSchema.ts',
```
[the remark this answers is not in the document]
Comment (gitstart-twenty) [reply packages/twenty-front/jest.config.js]: jest has issues parsing `zod` modules

[the remark this answers is not in the document]
Comment (gitstart-twenty) [reply packages/twenty-front/jest.config.js]: The `utilities/recoil-scope/utils/*` files primarily re-export Recoil methods with minimal wrappers. Testing these files might be redundant, as it resembles testing Recoil itself, a task that the Recoil team likely has already undertaken.

File: packages/twenty-front/jest.config.js [hunk 2]
```diff
@@ -33,7 +35,12 @@ export default {
     'generated-metadata/*',
     'generated/*',
     '__stories__/*',
-
+    'utilities/recoil-scope/utils/*',
```
[the remark this answers is not in the document]
Comment (gitstart-twenty) [reply packages/twenty-front/jest.config.js]: The `utilities/recoil-scope/utils/*` files primarily re-export Recoil methods with minimal wrappers. Testing these files might be redundant, as it resembles testing Recoil itself, a task that the Recoil team likely has already undertaken.

**Pull request merged**